### PR TITLE
A few small content changes 

### PR DESCRIPTION
**Pull request by EmmaFrith**
a few small content changes 

- to home page and linked sections to make page headings consistent with link content 
- to data requirements page to make language more human and reduce content length 

@edwardhorsford I couldn't figure out how to add text into a list on the data requirements page, so have formatted the text as a list item for now

### CODE REVIEW

File: app/views/data-requirements.html [hunk 1]
```diff
@@ -146,7 +148,7 @@ <h2 class='govuk-heading-s'>Non-UK degrees</h2>
         captionClasses: "govuk-visually-hidden",
         head: [
           {
-            text: "Specific data for Assessment Only",
+            text: "Specific data for assessment-only trainees",
```
Comment (edwardhorsford) [inline app/views/data-requirements.html]:
```suggestion
            text: "Specific data for assessment-only candidates",
```

I wonder if in this case we should use the correct term

File: app/views/data-requirements.html [hunk 2]
```diff
@@ -2,20 +2,18 @@
...
     <h1 class="govuk-heading-l">{{pageHeading}}</h1>
-    <p class="govuk-body">We only ask for the data we require to register trainee teachers. We have outlined what you need to collect for each route.</p>
-    <h2 class="govuk-heading-m">Contents</h2>
     <ul class="govuk-list govuk-list--dash govuk-!-margin-bottom-8">
-      <li><a class="govuk-link" href="#section-all-routes">Data required for all routes</a></li>
-      <li><a class="govuk-link" href="#section-assessment-only">Specific data for Assessment Only</a></li>
-      <li><a class="govuk-link" href="#section-provider-led">Specific data for Provider-led</a></li>
-    </ul> 
+      <li><a class="govuk-link" href="#section-all-routes">Standard data for all trainees</a></li>
+      <li><a class="govuk-link" href="#section-assessment-only">Specific data for assessment-only trainees</a></li>
```
Comment (edwardhorsford) [inline app/views/data-requirements.html]: Should we call them candidates here?

File: app/views/data-requirements.html [hunk 3]
```diff
@@ -81,7 +79,11 @@ <h2 class='govuk-heading-s'>Contact details</h2>
             {
               html: "
               <h2 class='govuk-heading-s'>Diversity information (optional)</h2>
-              <ul class='govuk-list'>
+              <ul class='govuk-list govuk-list--spaced'>
+                <li>
+                  If the trainee chose not to share diversity information during the application process, you should not ask them for it for the prupose of registering them.
+                </li>
+                <ul class='govuk-list'>
```
Comment (edwardhorsford) [inline app/views/data-requirements.html]: I don't think this ul should be here

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

**Review by edwardhorsford — APPROVED** [review]

File: app/views/data-requirements.html (hunk 1 above)
Comment (EmmaFrith) [reply app/views/data-requirements.html]: yes!

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**